contents/blog/dynamic-open-graph-images.md

```diff
@@ -0,0 +1,242 @@
+
+We also build custom artwork for pages we know are likely to be shared, like for an event we’ll be hosting soon.
+
+![Personalized Open Graph image for an event](../images/blog/open-graph-images/og-11.png)
+
+## **Dynamic Open Graph images**
+
+*Now for the fun part...*
+
+Similar to GitHub’s approach, we automatically generate some Open Graph images so our Graphic Designer, Lottie, doesn’t have to be on call 24x7 for whenever we publish a blog post.
+
+### Before/after: Sharing a `/docs` URL
+
+**Before (previous site-wide Open Graph image)**
+
+We shared a single static image across all Docs articles
+
+![Open Graph image](../images/blog/open-graph-images/og-0.png)
+
+**After**
+
+Now we show a preview from the article, along with some relevant information about it that you might like to know.
+
+![Open Graph image](../images/blog/open-graph-images/og-12.png)
+
+This gives us the chance to draw a user into the content on the page before they decide to click. Our bet was this could increase clickthroughs by reducing the uncertainty about what was on the other side of the link.
+
+Questions visitors may subconsciously have could include:
+
+- How fresh is this content?
+- How much of a time investment is this going to be?
+- Is this a website that's going to bombard me with ads?
+- Will I be able to easily access the information I might click to see?
+- Does this website "seem" like it will load quickly?
+
+Believe it or not, we can address all of these questions using visual cues in these images, and we can communicate them in a fraction of a second.
+
+### Other Open Graph templates throughout PostHog.com
+
+**Blog post**
+
+There are a considerations for our blog posts:
+
+- If we place an image in the background, text layered on top needs to be readable. The fix? We added subtle text shadows and partially transparent gradients layered between the text and image.
+- Some posts have multiple authors, so that needs to be factored in.
+- Some post images have text [inside the image](https://posthog.com/blog/yc-top-companies). In this case, we don't want to auto-generate an Open Graph image for these, we have a parameter in our blog post’s front matter to skip this image generation.
+
+![Blog Open Graph image](../images/blog/open-graph-images/og-13.png)
+
+**Customer stories**
+
+Since each customer story involves two logos, we placed them at the top along with a heart in between.
+
+![Example of Open Graph image in a customer story](../images/blog/open-graph-images/og-14.png)
+
+**Careers**
+
+Especially useful for the PostHog team when posting roles on LinkedIn, this image is always up to date with the latest roles we’re hiring.
+
+![Careers Open Graph image](../images/blog/open-graph-images/og-1.png)
+
+## How it works
+
+A more technical explanation of how we built this can be found [in this pull request](https://github.com/PostHog/posthog.com/pull/2695), but here’s a summary:
```

Comment:
The content in the PR is quite cool - I'd be tempted to copy paste it directly into this post.